spec/system/reporting/caseworker_report_spec.rb

```diff
@@ -1,23 +1,25 @@
 require 'rails_helper'
 
-RSpec.describe 'Caseworker Report' do
-  before do
-    visit '/'
-    visit reporting_user_report_path(:caseworker_report)
-  end
+RSpec.describe 'Caseworker report' do
+  include_context 'when viewing a temporal report'
+  let(:interval) { Types::TemporalInterval['daily'] }
+  let(:period) { '2023-01-01' }
 
-  context 'when logged in user is caseworker' do
-    it 'renders "Not found"' do
-      expect(page).to have_http_status(:not_found)
-    end
-  end
+  it 'includes the correct colgroup detail sortable headings' do
+    detail_headings = page.all('.app-table thead tr.colgroup-details th').map(&:text)
 
-  context 'when logged in as a supervisor' do
-    let(:current_user_role) { UserRole::SUPERVISOR }
+    expect(detail_headings).to contain_exactly(
+      ' ', 'byanother', 'closed', 'completed', 'fromanother', 'fromlist', 'fromself', 'sentback', 'total',
+      'total', 'total', 'unassigned'
+    )
+  end
 
-    it 'renders the caseworker report' do
-      expect(page).to have_text 'Caseworker report'
-      expect(page).to have_http_status(:ok)
+  it_behaves_like 'a table with sortable columns' do
+    let(:active) { ['user_name'] }
+    let(:active_direction) { 'ascending' }
+    let(:inactive) do
+      %w[percentage_closed_by_user percentage_unassigned_from_user total_assigned_to_user total_closed_by_user
```

Comment:
Perhaps put these on a new line for each item, I think it makes it easier to parse